### ISSUE TITLE
Java.NullPointerException crash

### ISSUE THREAD

**Issue by Rattlyy** [DaMatrix/Pork2b2tBot#27]
```
[06/11/2019 19:34:11] [Client] [ALERT] ****************************************
[06/11/2019 19:34:11] [Client] [ALERT] * 
[06/11/2019 19:34:11] [Client] [ALERT] * java.lang.NullPointerException
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.handler.incoming.entity.EntityHeadLookHandler.apply(EntityHeadLookHandler.java:30)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.handler.incoming.entity.EntityHeadLookHandler.apply(EntityHeadLookHandler.java:27)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.util.handler.HandlerRegistry.handleInbound(HandlerRegistry.java:63)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.ClientListener.packetReceived(ClientListener.java:50)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at com.github.steveice10.packetlib.event.session.PacketReceivedEvent.call(PacketReceivedEvent.java:51)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at com.github.steveice10.packetlib.tcp.TcpSession.callEvent(TcpSession.java:143)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at com.github.steveice10.packetlib.tcp.TcpSession$3.run(TcpSession.java:343)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at java.lang.Thread.run(Thread.java:748)
[06/11/2019 19:34:11] [Client] [ALERT] * 
[06/11/2019 19:34:11] [Client] [ALERT] ****************************************
[06/11/2019 19:34:11] [Client] [INFO] Disconnecting from server...
[06/11/2019 19:34:11] [Client] [INFO] Disconnected.
[06/11/2019 19:34:11] [Server] [INFO] Player disconnected: null
[06/11/2019 19:34:11] [Client] [ALERT] ****************************************
[06/11/2019 19:34:11] [Client] [ALERT] * 
[06/11/2019 19:34:11] [Client] [ALERT] * java.lang.NullPointerException
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.handler.incoming.entity.EntityHeadLookHandler.apply(EntityHeadLookHandler.java:30)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.handler.incoming.entity.EntityHeadLookHandler.apply(EntityHeadLookHandler.java:27)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.util.handler.HandlerRegistry.handleInbound(HandlerRegistry.java:63)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.ClientListener.packetReceived(ClientListener.java:50)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at com.github.steveice10.packetlib.event.session.PacketReceivedEvent.call(PacketReceivedEvent.java:51)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at com.github.steveice10.packetlib.tcp.TcpSession.callEvent(TcpSession.java:143)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at com.github.steveice10.packetlib.tcp.TcpSession$3.run(TcpSession.java:343)
[06/11/2019 19:34:11] [Client] [ALERT] * 	at java.lang.Thread.run(Thread.java:748)
[06/11/2019 19:34:11] [Client] [ALERT] * 
[06/11/2019 19:34:11] [Client] [ALERT] ****************************************
[06/11/2019 19:34:11] [ALERT] ****************************************
[06/11/2019 19:34:11] [ALERT] * 
[06/11/2019 19:34:11] [ALERT] * java.util.concurrent.ExecutionException: java.lang.NullPointerException
[06/11/2019 19:34:11] [ALERT] * 	at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:357)
[06/11/2019 19:34:11] [ALERT] * 	at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:1895)
[06/11/2019 19:34:11] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.PorkClientSession.getDisconnectReason(PorkClientSession.java:50)
[06/11/2019 19:34:11] [ALERT] * 	at net.daporkchop.toobeetooteebot.Bot.start(Bot.java:226)
[06/11/2019 19:34:11] [ALERT] * 	at net.daporkchop.toobeetooteebot.Bot.main(Bot.java:128)
[06/11/2019 19:34:11] [ALERT] * Caused by: java.lang.NullPointerException
[06/11/2019 19:34:11] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.handler.incoming.entity.EntityHeadLookHandler.apply(EntityHeadLookHandler.java:30)
[06/11/2019 19:34:11] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.handler.incoming.entity.EntityHeadLookHandler.apply(EntityHeadLookHandler.java:27)
[06/11/2019 19:34:11] [ALERT] * 	at net.daporkchop.toobeetooteebot.util.handler.HandlerRegistry.handleInbound(HandlerRegistry.java:63)
[06/11/2019 19:34:11] [ALERT] * 	at net.daporkchop.toobeetooteebot.client.ClientListener.packetReceived(ClientListener.java:50)
[06/11/2019 19:34:11] [ALERT] * 	at com.github.steveice10.packetlib.event.session.PacketReceivedEvent.call(PacketReceivedEvent.java:51)
[06/11/2019 19:34:11] [ALERT] * 	at com.github.steveice10.packetlib.tcp.TcpSession.callEvent(TcpSession.java:143)
[06/11/2019 19:34:11] [ALERT] * 	at com.github.steveice10.packetlib.tcp.TcpSession$3.run(TcpSession.java:343)
[06/11/2019 19:34:11] [ALERT] * 	at java.lang.Thread.run(Thread.java:748)
[06/11/2019 19:34:11] [ALERT] * 
[06/11/2019 19:34:11] [ALERT] ****************************************
[06/11/2019 19:34:11] [INFO] Shutting down...
[06/11/2019 19:34:11] [Server] [INFO] Closing server...
[06/11/2019 19:34:11] [Server] [SUCCESS] Server closed.
[06/11/2019 19:34:14] [WebSocket] [INFO] Shutting down...
[06/11/2019 19:34:14] [WebSocket] [SUCCESS] Shut down.

```

java -version:
`
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (build 1.8.0_222-8u222-b10-1~deb9u1-b10)
OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode)
`

Happened while using the bot as a proxy. Was using WWE client + Baritone process.

**Comment by DaMatrix** [comment]
hm, very interesting... not sure what could have caused that. were you actively playing (or was baritone active) when this happened?

**Comment by Rattlyy** [comment]
baritone was walking, i was watching yt then i enter minecraft and all crashes

**Comment by Rattlyy** [comment]
also add a thing that spams /queue main while in queue, it does this:
https://media.discordapp.net/attachments/635142261893759018/641700948527546382/unknown.png

**Comment by DaMatrix** [comment]
this should be fixed in fbea2e92c149acd815265d4693095a5abaf94adf, compiled build is available [here](https://maven.daporkchop.net/net/daporkchop/toobeetooteebot/0.2.3-1.12.2/toobeetooteebot-0.2.3-1.12.2.jar)
